docker/build_scripts/install-build-packages.sh

```diff
@@ -19,6 +19,9 @@ if [ "${AUDITWHEEL_POLICY}" == "manylinux2010" ] || [ "${AUDITWHEEL_POLICY}" ==
 elif [ "${AUDITWHEEL_POLICY}" == "manylinux_2_24" ]; then
 	PACKAGE_MANAGER=apt
 	COMPILE_DEPS="libz-dev libbz2-dev libexpat1-dev libncurses5-dev libreadline-dev tk-dev libgdbm-dev libdb-dev libpcap-dev liblzma-dev openssl libssl-dev libkeyutils-dev libkrb5-dev comerr-dev libidn2-0-dev libcurl4-openssl-dev uuid-dev libffi-dev linux-kernel-headers"
+elif [ "${AUDITWHEEL_POLICY}" == "musllinux_1_1" ]; then
+	PACKAGE_MANAGER=apk
+	COMPILE_DEPS="zlib-dev bzip2-dev expat-dev ncurses-dev readline-dev tk-dev gdbm-dev libpcap-dev xz-dev openssl openssl-dev keyutils-dev krb5-dev libcom_err libidn-dev curl-dev util-linux-dev libffi-dev linux-headers"
```

Comment:
I tried compiling a simple `c++`/`g++` hello world with this image.

It looks like `g++` is not fully set up yet:
```console
$ c++ main.cpp
ld: cannot find -lm
collect2: error: ld returned 1 exit status
```

I searched for a `libm` in the `/` dir and cannot find one. Adding `-static` also does not solve the problem.

Is something still missing in this base image? :)